src/HSV/HSVColorRange.php

```diff
@@ -27,11 +27,6 @@ public function containsIncl(HSVColor $hsvColor): bool
 
     public function containsExcl(HSVColor $hsvColor): bool
     {
-        return $hsvColor->hue        > $this->hueMin
-            && $hsvColor->hue        < $this->hueMax
-            && $hsvColor->saturation > $this->saturationMin
-            && $hsvColor->saturation < $this->saturationMax
-            && $hsvColor->value      > $this->valueMin
-            && $hsvColor->value      < $this->valueMax;
+        return ! $this->containsIncl($hsvColor);
```

Comment:
The "Incl" or "Excl" in the method names are meant to be understood as "Including boundary conditions" and "Excluding boundary conditions";

$hueMin = 1;
$hueMax = 3;

would mean with Incl that the number can be 1, 2 or 3, and with Excl only be 2.
As this already caused confusion I would accept a PR that remove the "containsExcl" and renames the "containsIncl" to "contains" instead.

Reply:
Done in https://github.com/PrinsFrank/colors/commit/99b5c15afbd4160331159a1c1206e0624dc72faa